### PR TITLE
WDB-49/Multiple-video-delete

### PR DESCRIPTION
**Pull request by povilassta**
Small PR that refactors video delete endpoint to support multiple video removal. Also changes API call in frontend to support changes in backend.

### CODE REVIEW

File: backend/RestAPI/Controllers/VideosController.cs [hunk 1]
```diff
@@ -192,25 +192,33 @@ public async Task<IActionResult> DeleteVideo([FromRoute] Guid id)
...
+                    return NotFound();
+                }
+
+                try
+                {
+                    await _videoService.DeleteVideo(video, user.Id);
+                }
+                catch (FileNotFoundException ex)
+                {
+                    return BadRequest(ex.Message);
```
Comment (tmaciulis22) [inline backend/RestAPI/Controllers/VideosController.cs]: Return NotFound(ex.Message) here

**Review by tmaciulis22 — APPROVED** [review]
🔍 Minor change, but overall very good job 👍🏻

**Review by kibartas — APPROVED** [review]

File: backend/RestAPI/Controllers/VideosController.cs [hunk 2]
```diff
@@ -192,25 +192,33 @@ public async Task<IActionResult> DeleteVideo([FromRoute] Guid id)
...
-                await _videoService.DeleteVideo(video, user.Id);
-            }
-            catch (FileNotFoundException ex)
-            {
-                return BadRequest(ex.Message);
-            }
-            catch
-            {
-                return StatusCode(500);
+                var video = await _videosRepository.GetVideoById(id);
```
Comment (kibartas) [inline backend/RestAPI/Controllers/VideosController.cs:199]: Same as for Aurimas, should we be calling a repository in controller

Comment (tmaciulis22) [reply backend/RestAPI/Controllers/VideosController.cs:199]: It is totally fine to do that. @povilassta You can merge this PR

**Pull request merged**